src/Weasel.Postgresql.Tests/IntegrationContext.cs

```diff
@@ -37,17 +37,19 @@ protected async Task ResetSchema()
     protected async Task CreateSchemaObjectInDatabase(ISchemaObject schemaObject)
     {
         var rules = new PostgresqlMigrator();
-        var writer = new StringWriter();
-        schemaObject.WriteCreateStatement(rules, writer);
+        var builder = new DbCommandBuilder(theConnection);
```

Comment:
As I mentioned earlier. I'd like to see a precise explanation of why we need to change the base class for all tests to enable verification of your changes. I'm not against it if needed, but I'd prefer not to do it, as this is an invasive change that can cause regression issues.

Reply:
The main reason to change this is that the new version includes changes that were made in PostgresqlMigrator.cs and thus allows creating concurrent indexes in tests as well. I think it can be moved to a separate method, but as I see it does just the same thing as the previous version.
Regarding regression I can't see any issues as all tests are passed. I'm looking forward to your suggestions how to improve it.

Reply:
@oskardudycz Does that answer your question? Maybe I just miss something. I look forward to your thoughts.

Reply:
Ok, fine by me :+1: